docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,103 @@
+# Tizen 6.0
+
+The second milestone (M2) release of Tizen 6.0 was announced in October 2020.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+
+The key features of Tizen 6.0 are the enhancement of the development environment, core, and AI components.
+
+## Enhancement of Development Environment and Core Component
+
+### Tizen.NET
+
+Tizen supports C# development environment based on .NET Core for application development productivity.
+
+In Tizen 6.0, the latest  .NETCore 3.1 and the latest version of cross-platform Xamarin are supported. With this update, there is a considerable advancement in the application start-up time and reduction in memory usage. The **XAML Hot Reload** beta service is supported as well.
+
+The following is updated in Tizen 6.0:
+
+- **.NET Core**:
+     - Latest .NET Core 3.1(LTS)
+     - Optimized performance
+     - Diagnostics tools
+- **TizenFX**:
+    - Tizen platform-specific API set
+    - API level 8 released
+- **Xamarin**:
+    -  Latest Xamarin.Forms (4.8.0)  and Xamarin.Essentials (1.5.3) are available.
+    - TV UI Controls and CircularUI: New UI controls for Smart TVs and Smart Watches.
+    - XAML Hot Reload: Boosting developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device various offloading services
+
+As the complexity of applications in the web environment increases, a method of maximizing computational power is needed.
+
+Tizen 6.0 provides **multi-device offloading service** technology using multiple devices.
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES 10 up to a large extent:
+
+- For more information about **Lightweight JavaScript engine**, see [ESCARGOT](https://github.com/Samsung/excargot). The following are the details for memory size and features in comparison to Chrome V8 benchmarks:
```

Comment:
The link is breaking now, please verify the address. https://github.com/Samsung/excargot

Reply:
Thanks for checking it. I think we don't need to provide the detail info for this. so I removed.  I will commit it after removing.